docs/package.json

```diff
@@ -4,7 +4,9 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
+    "prestart": "cd ../cli; make docgen;",
```

Comment:
This explains why this works:

https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts